index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+  <script src="https://unpkg.com/boxicons@2.1.4/dist/boxicons.js"></script>
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <i class='bx bx-menu'></i>
+      <!--<img src="images/icons/Union.svg"> -->
+    </nav>
+</header>
+  <section class="headline">
+    <div class="container">
+      <h2>Hey there. <br>
+        I’m Tsige.</h2>
```

Comment:
Kindly use the `h1` tag here to enable google search algorithm to detect the main title of the page 😄